test/test_lib.py

```diff
@@ -169,19 +161,23 @@ def test_namespace(self):
         assert list(ops.lib._find_all_specs([tmpdir])) == []
 
 
-class TestLibParser(TestCase):
-    def _mkmod(self, name: str, content: typing.Optional[str] = None) -> ModuleSpec:
+class TestLibParser:
+    def _mkmod(
+        self,
+        request: pytest.FixtureRequest, name: str,
+        content: typing.Optional[str] = None
```

Comment:
Per comment in the other PR, let's standardise on trailing comma here and in similar places, to avoid extra lines in future diffs:
```suggestion
        content: typing.Optional[str] = None,
```